conf/scylla.yaml

```diff
@@ -221,9 +220,6 @@ rpc_address: localhost
 # rpc_interface: eth1
 # rpc_interface_prefer_ipv6: false
 
-# port for Thrift to listen for clients on
-rpc_port: 9160
```

Comment:
The fact we still have rpc_address but get rid of rpc_port as an absolute user experience disaster, but it's not a new one so let's not fix it right now :-(

Reply:
@nyh i don't follow you. IIUC, this file serves as a sample/template configuration. if we remove an option that's not used any more from it, will user be confused? if we keep it, i think it would be more confusing, no?

Reply:
One of the reasons why I wrote docs/dev/protocols.md in the first place is this word "rpc" which confused me **every time** - sometimes we use this word RPC to refer to inter-node communication, sometimes we use it to refer to Thrift, and sometimes, like "rpc_address", it even refers to CQL. This is super confusing, what I called a "absolute user experience disaster", and I just noted that it's sad we didn't completely get rid of this mess. As I noted, we got rid of "rpc_port" in this patch, but "rpc_address" remains. It should have been called "cql_address" and everything will be much clearer.
But as I said, we don't need to fix this right now, and perhaps we can't even fix it completely (for backward compatibility) but perhaps we can consider in the future adding an alias or something.

Reply:
@nyh so, IIUC, it's fine removing it from `scylla.yaml`.  but yeah, the symmetry and correctness of the options are still problems that we need to address.